projects/plugins/jetpack/modules/masterbar/admin-menu/class-dashboard-switcher-tracking.php

```diff
@@ -0,0 +1,167 @@
+<?php
+/**
+ * Quick switcher tracking file.
+ *
+ * @package automattic/jetpack
+ */
+
+namespace Automattic\Jetpack\Dashboard_Customizations;
+
+use Automattic\Jetpack\Terms_Of_Service;
+use Automattic\Jetpack\Tracking;
+use Jetpack_Plan;
+
+/**
+ * Class Dashboard_Switcher_Tracking
+ */
+class Dashboard_Switcher_Tracking {
+
+	/**
+	 * Jetpack Tracking library will prefix the event name with "jetpack_*" automatically.
+	 */
+	const JETPACK_EVENT_NAME = 'dashboard_quick_switch_link_clicked';
```

Comment:
I could've  added the wpcom_ prefix automatically but according to FG, we should avoid generating them dynamically to improve search-ability.